File: src/main/java/co/edu/upb/trenes/models/trenes/NodoVagon.java
package co.edu.upb.trenes.models.trenes;

public class NodoVagon {
    private Vagon vagon;
    private NodoVagon siguiente;

    public NodoVagon(Vagon vagon) {
        this.vagon = vagon;
    }

    public Vagon getVagon() { return vagon; }
    public NodoVagon getSiguiente() { return siguiente; }
    public void setSiguiente(NodoVagon siguiente) { this.siguiente = siguiente; }
}

File: docs/legacy-csharp/Gestion Trenes y Vagones/NodoVagon.cs
internal class NodoVagon
{

        private Vagon vagon;
        private NodoVagon siguiente;

        public Vagon Vagon { get => vagon; set => vagon = value; }
        public NodoVagon Siguiente { get => siguiente; set => siguiente = value; }

        public NodoVagon(Vagon vagon)
        {
            Vagon = vagon;
            Siguiente = null;
        }

    }
